src/lib/farming/types.ts

```diff
@@ -0,0 +1,27 @@
+export enum FarmingPatchTypes {
+	Herb = 'herb',
+	FruitTree = 'fruit tree',
+	Tree = 'tree',
+	Allotment = 'allotment',
+	Cactus = 'cactus',
+	Bush = 'bush',
+	Flower = 'flower',
+	Spirit = 'spirit',
+	Hardwood = 'hardwood',
+	Seaweed = 'seaweed',
+	Vine = 'vine',
+	Calquat = 'calquat',
+	Redwood = 'redwood',
+	Crystal = 'crystal',
+	Celastrus = 'celastrus',
+	Hespori = 'hespori'
+}
+
+export interface PatchData {
+	lastPlanted: string;
+	patchStage: boolean; // false -> nothing planted, true -> something planted
```

Comment:
is it really the "stage"  of the patch to say whether there's no patch, or that its planted?

the naming doesnt make sense to me.